exporter/awsxrayexporter/conn.go

```diff
@@ -53,6 +56,11 @@ const (
 	STSAwsCnPartitionIDSuffix = ".amazonaws.com.cn" // AWS China partition.
 )
 
+var collectorUserAgentHandler = request.NamedHandler{
+	Name: "otel.collector.UserAgentHandler",
+	Fn:   request.MakeAddToUserAgentHandler("opentelemetry-collector-contrib", version.Version, version.GitHash),
```

Comment:
Can we also have a way to customize the values for user agent? I hope we can override the collector name and version value by reading values form env var.

Reply:
I went ahead and added the `distributionType` variable in this PR so that would be modifiable with ldflags